lua/entities/starfall_processor/shared.lua

```diff
@@ -137,30 +137,15 @@ function ENT:Error(err)
 	if newline then
 		msg = string.sub(msg, 1, newline - 1)
 	end
-	if self.owner:IsValid() then
-		SF.AddNotify(self.owner, msg, "ERROR", 7, "ERROR1")
-	end
+
+	hook.Run("StarfallError", self, self.owner, CLIENT and LocalPlayer() or false, self.sfdata.mainfile, msg, traceback)
+	SF.SendError(self, msg, traceback)
```

Comment:
Either check whether the `chip.owner` is valid here *(could've disconnected)*, or allow `StarfallError` to run with an invalid owner. I think the latter is more desirable. Just need to remember to check for owner parameter validity when using that hook later.